CMakeLists.txt

```diff
@@ -78,7 +79,8 @@ message("QUAD_PRECISION ... ${QUAD_PRECISION}")
 message("REPRO ............ ${REPRO}")
 message("WW3 .............. ${WW3}")
 message("S2S .............. ${S2S}")
-message("DATM ............. ${DATM}")
+message("CDEPS ............ ${CDEPS}")
```

Comment:
What is the difference here between CDEPS and CDEPS_DATM?
If it is DATM, it is expected to be CDEPS, correct? So why the distinction?

Reply:
DATM is the the existing data component. I think we were plaining to keep both CDEPS and DATM together for a while. @aerorahul CDEPS is the main component that serves all the data component such as ocean, atmosphere and wave. This flag able to build CDEPS itself (all data components). While, CDEPS_DATM activate CDEPS provided data atmosphere, it just link libdatm to the model. So, if you want to activate DATM and DOCN together, you need to provide CDEPS=Y CDEPS_DATM=Y and CDEPS_DOCN=Y

Reply:
Since the old NEMS "DATM" will be retired - I suggest that DATM, DOCN, DICE, etc. always refer to CDEPS components.  The existing DATM could be renamed DATM_NEMS if it needs to be retained.

Reply:
@rsdunlapiv That is exactly what my comment meant.

Reply:
My understanding is the current plan is to commit this with DATM meaning the existing NEMSDatm; the CDEPS DATM is brought in using CDEPS_DATM. Eventually DATM (meaning original NEMSDatm) will be removed. We will still need the CDEPS_DATM because CDEPS itself can be handling a data ocean or atmosphere.

Reply:
I don't think we need CDEPS, CDEPS_DATM and CDEPS_DOCN and so on.
Since all Data components will be CDEPS, that is the only CDEPS we would need.
Irrelevant to the Data application (DATM or DOCN), CDEPS will be ON.

Reply:
@DeniseWorthen , You are correct. CDEPS is used for shared libraries (such as fox and those built from share,..) used by CDEPS_DATM and future CDEPS_DOCN, CDEPS_DWAV, etc. DATM is the NEMSdatm component. 

In the future, when DATM is removed from ufs-weather-model, CDEPS_DATM will be changed to DATM.

Reply:
@aerorahul Right now, CDEPS_DATM is used in NEMS/src/module_EARTH_GRID_COMP.F90 (line 74) for building the coupled mom6_cice6_cdeps_datm application.

Reply:
That is an `#ifdef` in the code.  How is that relevant to the cmake variable here?
Why can't we in this PR decide that DATM is CDEPS and define a variable `DATM_NEMS` and gut it subsequently.
The next PR then becomes simpler in removing NEMS based DATM and that's it.

Like @rsdunlapiv suggested, we should have a no frills PR which adds CDEPS and then DATM is understood to be CDEPS, unless noted as DATM_NEMS.

Reply:
@aerorahul I will revise CMakeLists.txt to make the changes.

Reply:
I think the purpose of this message is to just say what was built.  That is different from the configuration that is actual run.

Reply:
@rsdunlapiv Thanks. I'm sure where my comment belongs, but I was trying to indicate that at the application level, we will still need something to specify which component CDEPS is standing in for---atm, ocn or ice.